### DISCUSSION TITLE
buddhism/2024/level_1/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#83] [General]
# buddhism/2024/level_1/

https://shanenull.com/buddhism/2024/level_1/

**Comment by shane0** [comment]
First ten weeks
https://shanenull.com/buddhism/2023/level_1/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2024/samudaya/

**Comment by shane0** [comment]
Format option daily outside class
https://shanenull.com/buddhism/2023/record/

**Comment by shane0** [comment]
# 2024 calendar

## 01

The Buddhadharma teaches the four noble truths: the existence of suffering, the causes of suffering, the cessation of suffering (nirvana), and the path to the cessation of suffering. It is said, "Recognize suffering and remove its causes; aspire to nirvana and practice the path." We must first know what suffering is, then remove its underlying causes. Aspiring to the virtues of nirvana - permanence, ultimate joy, the true self, and intrinsic purity,
we practice the path to eradicate afflictions and attain nirvana. This is life's highest truth.
- Words of Wisdom from Grand Master Weichueh

## 02

Bringing forth the mind is the most important aspect of Buddhadharma. This very mind is like a bell. Powerful strikes bring strong tones, gentle strikes create soft tones, but with no strike there can be no sound. Ring the bell once, and it will sound once. Do not ring the bell, and it hangs in complete silence. To strike is to ring, but who is the one ringing the bell? The mind-bell cannot be rung by others, only by oneself. As one rings the mind-bell, one brings forth the mind.
- Words of Wisdom from Grand Master Weichueh


## 03

Shamatha is halting the mind in stillness, fixing it on one place. Vipashyana is maintaining this very mind in a state of clarity and lucidity. Reflect inward to examine the afflictions of greed, anger, and ignorance in our minds -which one is gravest? Then contemplate these afflictions in the samadhi achieved through shamatha, and remove them with the wisdom obtained from vipashyana. Practicing shamatha without vipashyana only brings a state of vacant stillness, where we cannot generate wisdom or remove our afflictions. Therefore, shamatha and vipashyana must be applied simultaneously.
- Words of Wisdom from Grand Master Weichueh

**Comment by shane0** [comment]
# WeiChueh

- dharma name: only awareness
- https://sunnyvale.ctzen.org/wiki/about-us/chung-tai-chan-monastery/
- Grand Master WeiChueh applies these three concepts: 
  - Three Links of Cultivation
  - The Four Tenets of Chung Tai
  - The Five Approaches of Propagation Modern Buddhism
